.github/workflows/run-test-manual-new.yml

```
@@ -0,0 +1,186 @@
+          SHARD: ${{ matrix.shard }}
+          BASE_URL: ${{ needs.initialize-cloud.outputs.BASE_URL }}
+          API_URL: ${{ needs.initialize-cloud.outputs.API_URL }}
+          E2E_USER_NAME: ${{ secrets.E2E_USER_NAME }}
+          E2E_USER_PASSWORD: ${{ secrets.E2E_USER_PASSWORD }}
+          E2E_PERMISSIONS_USERS_PASSWORD: ${{ secrets.E2E_PERMISSIONS_USERS_PASSWORD }}
+          E2E_ENCODE_PASS: ${{ secrets.E2E_ENCODE_PASS }}
+          ACCOUNTS: ${{ needs.initialize-cloud.outputs.ACCOUNTS }}
+          MAILPITURL: ${{ secrets.MAILPITURL }}
+          PW_WORKERS: ${{ vars.PW_WORKERS }}
+          PW_RETRIES: ${{ vars.PW_RETRIES }}
+          ENABLED_SERVICE_NAME_HEADER: true
```

actionlint (1.7.4)

input "ENABLED_SERVICE_NAME_HEADER" is not defined in action "Run Playwright tests" defined at "./.github/actions/run-pw-tests". available inputs are "ACCOUNTS", "API_URL", "BASE_URL", "BRANCH_NAME", "DASHBOARD_VERSION", "E2E_ENCODE_PASS", "E2E_PERMISSIONS_USERS_PASSWORD", "E2E_USER_NAME", "E2E_USER_PASSWORD", "MAILPITURL", "PROJECT", "PW_RETRIES", "PW_WORKERS", "SALEOR_CLOUD_SERVICE", "SHARD", "URL_TO_RUN"

(action)